### DISCUSSION TITLE
The wallet was successfully linked, but the user was very slow when purchasing the call contract, slow and the same type of item

### DISCUSSION THREAD

**Discussion by robotmeme-crypto** [WalletConnect/walletconnect-monorepo#3797] [Developer Support]
Successful link wallet, but the user in the purchase of tokens to call the contract, very slow, slow and the same type of project, what is the reason?

**Comment by glitch-txs** [comment]
Hi, thanks for reporting. Please read [Issues & Discussions guides](https://github.com/orgs/WalletConnect/discussions/3457) and kindly provide an MRE so we can help you with this issue. Thanks!
